### DISCUSSION TITLE
Feature Request: add support for Polars dataframes, in addition to pandas

### DISCUSSION THREAD

**Discussion by DeflateAwning** [zauberzeug/nicegui#3280] [Ideas / Feature Requests]
Polars is quickly displacing pandas. Would be awesome to see first-class support for it!

**Comment by falkoschindler** [comment]
That's right, @DeflateAwning!
We'd be happily accepting a PR for this feature. I guess it should be rather straight-forward to add `from_polars` methods to `ui.table` and `ui.aggrid`, in analogy to pandas.

**Comment by evnchn** [comment]
We got https://github.com/zauberzeug/nicegui/pull/3969 already, can close?
